CHANGELOG.md

```diff
@@ -115,6 +115,7 @@ Release report: https://github.com/wazuh/wazuh/issues/15504
 
 ### Fixed
 
+- Enhancing the handling of authd and remoted simulators in case of restart failures ([#Wazuh-jenkins#3487](https://github.com/wazuh/wazuh-qa/pull/4205)) \- (Tests)
```

Comment:
4.4.0 has already been released, so this change will not go into this version. The next version is 4.4.4.

Reply:
Changed in [fe11e5f](https://github.com/wazuh/wazuh-qa/pull/4205/commits/fe11e5ffe82b1ff89533d450a0a2a0745ff8e1cc)

Reply:
Fix the conflicts, please.

Reply:
Done in fe11e5f